guildwars2/notifiers.py

```diff
@@ -875,11 +903,11 @@ async def news_checker(self):
     async def before_news_checker(self):
         await self.bot.wait_until_ready()
 
-    @tasks.loop(minutes=1)
+    @tasks.loop(seconds=10)
```

Comment:
Probably no need to change that. Hitting them up too often risks cloudflare blocking something again, I think